Makefile

```diff
@@ -11,7 +11,10 @@ cmds = $(shell ls cmd)
 all: build
 
 format:
+	@echo "Format Go code"
 	@report=`$(GOFMT) -s -d -w $$(find cmd pkg test -name \*.go)` ; if [ -n "$$report" ]; then echo "$$report"; exit 1; fi
+	@echo "Clean up unused Go modules"
+	@report=`$(GO) mod tidy -v 2>&1` ; if [ -n "$$report" ]; then echo "$$report"; exit 1; fi
```

Comment:
I'm not sure this is the best place to put this. would 'clean' target be a bit better?

Reply:
`clean` is about removing build artifacts. `format` is for the stuff that ends up in git.

Reply:
FWIW, I'd prefer `format` more than `clean`.